pytripgui/view/qt_view_adapter.py

```diff
@@ -172,6 +172,14 @@ def checked_items(self):
                 selected.append(widget.data(Qt.UserRole))
         return selected
 
+    def check_items(self, items, lambda_names):
+        to_be_checked = [
```

Comment:
to be checked sounds like be checked by some external tool, maybe we use word selected ? or checkbox selected ?

Reply:
or maybe `items_to_check`?

Reply:
you can select item without checking it, so those words are not interchangeable in that case

Reply:
we should be explicit then, like `toogle_checkbox_for_items` or `set_checkbox_enabled` or  `set_tickbox` ?

Reply:
Done